src/validation/decorators/is_kolibri_trigger_n.ts

```diff
@@ -27,14 +27,18 @@ import {
 } from 'class-validator';
 
 
+export function isKolibriNodeTriggerN(value: any) {
+    if (!isBoolean(value) && !isString(value) && !isNumber(value)) {
```

Comment:
Isnt trigger mode just a number? Why checking if its a boolean and string?

Reply:
Totally right! TriggerN can only be a js type number.